lua/browse.lua

```diff
@@ -12,7 +12,7 @@ local build_prefixes = { "dev%-", "preprod%-", "" }
 --
 local function is_promoted(build_name)
   for i=1, #build_prefixes do
-    if build_name:match("^[a-z]+:[a-z]+:[%-A-Za-z0-9]+:" .. build_prefixes[i] .. "promoted%-") then
+    if build_name:match("^[a-z]+:[%-a-z]+:[%-A-Za-z0-9]+:" .. build_prefixes[i] .. "promoted%-") then
```

Comment:
This is a trival (and unrelated) fix that does relax the regexp in order to match some scalanga-devl content on GCS dev env.